### PR TITLE
fixes to be able to release and do heroku stuff

### PR DESCRIPTION
**Pull request by alex-bezek**
### Summary
There are some issues with prop types and heroku deploys in our app currently. These changes clean up our navigation prop types and also allow us to run the prop-types table generator, along with deploy to heroku

Please add your name to the [CONTRIBUTORS.md] file. Adding your name to the [CONTRIBUTORS.md] file signifies agreement to all rights and reservations provided by the [License].

Thanks for contributing to Terra. 
@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,10 +42,10 @@
     "clean:install": "npm run clean:all && npm install",
     "clean:obsolete-snapshots": "npm test -- -u",
     "compile": "lerna run compile",
-    "compile:heroku": "cd packages/terra-consumer-site && npm run compile:heroku",
+    "compile:heroku": "cd packages/terra-site && npm run compile:heroku",
```
Comment (bjankord) [inline package.json]: keep this as `terra-consumer-site`

Comment (alex-bezek) [inline package.json]: updated

**Comment by zhongyn** [comment]
+1

File: packages/terra-consumer-nav/src/Nav.jsx [hunk 2]
```diff
@@ -5,7 +5,6 @@ import Button from 'terra-button';
 import IconClose from 'terra-icon/lib/icon/IconClose';
```
Comment (amf9t2) [inline packages/terra-consumer-nav/src/Nav.jsx]: there is not test refactored after this change ?

Comment (alex-bezek) [inline packages/terra-consumer-nav/src/Nav.jsx]: no

**Pull request merged**